documentation/milestones/individual_milestones/search_mvp.md

```diff
@@ -0,0 +1,210 @@
+- Filter the search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- Sort the search results based on the value of *at least*
+
+Through the design and development of this API endpoint, the team should answer the following questions and record the decisions in one or more ADRs:
+
+- How should search and filter criteria be passed to the API endpoint used for search?
+- How should we support basic logic such as greater than, less than, as well as and/or/not conditions?
+- Which fields should support filtering and sorting?
+- Do we use an existing search service (e.g. Elasticsearch) or build our own?
+
+### Search Page
+
+The primary page we'll need to deliver for the user interface (UI) is a search page, which should allow users to:
+
+- Enter key words to search the list of funding opportunities
+- Filter search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- View a common set of fields for each opportunity returned by the search (e.g. title, award date, summary, etc.)
+- Sort the set of results returned by *at least* one structured field (e.g. award date, title, etc.)
+
+### Definition of done
+<!-- Required -->
+
+- [ ] Basic requirements:
+  - [ ] Code is deployed to `main` & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] Simpler site translations are live in PROD (if necessary). Translations of the search results content coming from grants.gov will not be included in this 30k ft deliverable. 
+  - [ ] All new services have passed a security review (if necessary)
+  - [ ] All new services have completed a 508 compliance review (if necessary)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly
+- [ ] Functional requirements:
+  - [ ] Users can search for opportunities by key word
+  - [ ] Users can filter search results by *at least one* structured field (e.g. award date, opportunity type, etc.)
+  - [ ] Users can sort search results by *at least* one structured field (e.g. award date, title, etc.)
+  - [ ] All of the search features available in the UI are also available via the API, and vice versa
+  - [ ] Search criteria are reflected in the URL so that users can bookmark or share the link to a given set of search results
+  - [ ] Users can access the corresponding grants.gov page for an opportunity they find on simpler.grants.gov via link on the search results
+
+### Proposed metrics for measuring goals/value/definition of done
+<!-- Required -->
+
+1. Total number of searches made via the UI
+2. Total number of searches made via the API
+3. Number of results returned per search
+4. Number of searches made per search term
+5. Number of searches made per filter combination
+6. Click conversion rate from search results to the opportunity listing (on grants.gov)
```

Comment:
Eventually it would be great to have metrics about whether the highest-ranked results using our algorithm are the "right" results for users. Like are they clicking on the first result based on their search. 

There might be some good articles or blogs about how search engines like Google evaluate the effectiveness of their search results.

And having supervised learning based on "was this relevant to your search? yes/no".

But those are complex measures, so maybe we defer them to Search 2.0.

Reply:
Good call, I added a note that we may want to track search result accuracy in the future in [this commit](https://github.com/HHS/simpler-grants-gov/pull/579/commits/0a05c4a125042018476cf6e64e3418b935b1fcee)